legal-api/src/legal_api/services/document_meta.py

```diff
@@ -386,12 +386,17 @@ def create_report_object(self, title: str, filename: str, report_type=None):
             'filename': filename
         }
 
-    def get_general_filename(self, name: str):
+    @staticmethod
+    def general_filename(filing_date: str, business_identifier: str, name: str) -> str:
         """Return a general filename string."""
-        filing_date_str = LegislationDatetime.format_as_legislation_date(self._filing_date)
-        file_name = f'{self._business_identifier} - {name} - {filing_date_str}.pdf'
+        filing_date_str = LegislationDatetime.format_as_legislation_date(filing_date)
+        file_name = f'{business_identifier} - {name} - {filing_date_str}.pdf'
```

Comment:
I would suggest not to use document_meta.py reference in v2 (this file is only used in v1), if needed you can create this function in `v2/business/business_documents.py` and you may not need file name in this format. I was thinking that we should show the name (`Certified Rules`) as same as the document list, but can you confirm this with UX team.

Reply:
We had some other requirements that have crept up:

1. If there is a memorandum change, do not generate the memorandum output
2. If the rules have been updated through SR - we require the name: Rules - Uploaded_Rules.pdf  - in the ledger

![image](https://user-images.githubusercontent.com/3484109/235985228-aa852c12-a238-4de8-89e2-d783e9a7c969.png)


@ethantspitt is the UX designer for this and wanted this done after feedback from the business and discussing with Janis